FantasmoSDK/app/src/main/java/com/fantasmo/sdk/config/RemoteConfig.kt

```diff
@@ -13,22 +16,39 @@ import java.lang.StringBuilder
 class RemoteConfig {
 
     data class Config(
+        @SerializedName("remote_config_id")
         var remoteConfigId: String,
+        @SerializedName("frame_acceptance_threshold_timeout")
         var frameAcceptanceThresholdTimeout: Float,
+        @SerializedName("is_behavior_requester_enabled")
         var isBehaviorRequesterEnabled: Boolean,
+        @SerializedName("is_tracking_state_filter_enabled")
         var isTrackingStateFilterEnabled: Boolean,
+        @SerializedName("is_movement_filter_enabled")
         var isMovementFilterEnabled: Boolean,
+        @SerializedName("movement_filter_threshold")
         var movementFilterThreshold: Float,
+        @SerializedName("is_blur_filter_enabled")
         var isBlurFilterEnabled: Boolean,
+        @SerializedName("blur_filter_variance_threshold")
         var blurFilterVarianceThreshold: Float,
+        @SerializedName("blur_filter_sudden_drop_threshold")
         var blurFilterSuddenDropThreshold: Float,
+        @SerializedName("blur_filter_average_throughput_threshold")
         var blurFilterAverageThroughputThreshold: Float,
+        @SerializedName("is_camera_pitch_filter_enabled")
         var isCameraPitchFilterEnabled: Boolean,
+        @SerializedName("camera_pitch_filter_max_upward_tilt")
         var cameraPitchFilterMaxUpwardTilt: Float,
+        @SerializedName("camera_pitch_filter_max_downward_tilt")
         var cameraPitchFilterMaxDownwardTilt: Float,
+        @SerializedName("is_image_quality_filter_enabled")
         var isImageQualityFilterEnabled: Boolean,
+        @SerializedName("image_quality_filter_score_threshold")
         var imageQualityFilterScoreThreshold: Float,
+        @SerializedName("image_quality_filter_model_uri")
         var imageQualityFilterModelUri: String?,
```

Comment:
Just wondering why we are using null safety here? I think we sort of want to error (well, to handle the decode error) if any value returned by the remote config fails to decode. Sort of brings it out of line with all the other variables, does it not?

Reply:
There is extra logic for these parameters further down in the model interpreter, also these fields are not in the default config which would then fail to load